src/components/general/ErrorMessage/index.tsx

```diff
@@ -0,0 +1,103 @@
+import * as React from "react";
+import Button from "../Button";
+import styles from "./styles.less";
+import * as Icons from "./errorIcons";
+import classNames from "classnames";
+
+export enum ErrorTypes {
+    error= "error",
+    accessDenied= "accessDenied",
+    notFound= "notFound",
+    noData= "noData",
+    noAttachment= "noAttachment",
+    noActionsCreated= "noActionsCreated",
+}
+
+type ErrorMessageProps = {
+    hasError?: boolean,
+    errorType?:ErrorTypes,
+    message?: any,
+    resourceName?: string,
+    title?: string,
+    children?: any,
+    icon?: any,
+    button?: string,
```

Comment:
Could this be something a bit more descriptive than "button"? "action" or something like that?

Reply:
Changed to action